### PR TITLE
Added sample data and implemented load on startup

### PR DESCRIPTION
**Pull request by ewilson72**
Can someone test this?  I can't run the emulator on my laptop.  It should print to terminal either success or an exception message

### CODE REVIEW

File: app/src/main/res/raw/SampleData.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿Key,Name,Latitude,Longitude,Street Address,City,State,Zip,Type,Phone,Website
```
Comment (danielprofili) [inline app/src/main/res/raw/SampleData.csv]: I get an error on build that says res/raw files can't be uppercase but renaming to `sampledata.csv` worked

Comment (danielprofili) [inline app/src/main/res/raw/SampleData.csv]: res/raw files can't have uppercase apparently (it wouldn't build) but renaming to `sampledata.csv` worked for me

File: app/src/main/java/kantwonskids/donationtrackerg14b/controller/WelcomeActivity.java [hunk 2]
```diff
@@ -27,5 +35,66 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                        state,
+                        zipcode,
+                        type,
+                        phoneNumber,
+                        website
+                );
+                model.donationDataList.add(d);
+            }
+            // print the success
+            System.out.print("Successfully populated the donationDataList");
```
Comment (danielprofili) [inline app/src/main/java/kantwonskids/donationtrackerg14b/controller/WelcomeActivity.java]: I don't think you can println() in android, you have to do `Log.d()` I think

**Review by danielprofili — CHANGES_REQUESTED** [review]
After changing two things I got a success message and after debugging to see the donation data list it appears to have worked!

**Review by AidanMulaokar — DISMISSED** [review]

**Review by aschmitt8 — COMMENTED** [review]
looks good! got it to work after making the two changes daniel mentioned (changing system.out.println to log.d and changing the name of SampleData.csv to sampledata.csv). so i'll approve once those changes are made

**Comment by ewilson72** [comment]
fixed the stuff

**Review by AidanMulaokar — DISMISSED** [review]

**Review by aschmitt8 — APPROVED** [review]

**Review by danielprofili — APPROVED** [review]

**Review by AidanMulaokar — APPROVED** [review]

**Pull request merged**